### PR TITLE
seven_seg.h

### PR DESCRIPTION
**Pull request by msorce26**
Add seven_seg.h file.

### CODE REVIEW

File: include/seven_seg.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ * What Module Does: This module provides functions to populate up to four seven segment displays.
+ * Available functions allow for four displays to be populated based on a singular 16-bit uint16_t input
+ * as well as 4 individual 8-bit uint8_t inputs.
+ * 
+ * How To Use Module: Call SevenSeg_Init() to initialize the module.
+ * Once initialized, calling SevenSeg_DisplayHex() or SevenSeg_DisplayRaw()
+ * will populate four seven segment displays based on the function inputs.
+ *
+ * @Authors: Michael Sorce, Alex Marino, Michael Maloney, and David Gaffney
```
Comment (muhlbaier) [inline include/seven_seg.h]: @Authors doesn't exist in Doxygen, use @author multiple times for each author

File: include/seven_seg.h [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+/**
+ * @defgroup seven_seg Seven Seg Module
+ * @brief The Seven Seg Module allows for values to be displayed on a seven segment display
+ *
+ * What Module Does: This module provides functions to populate up to four seven segment displays.
+ * Available functions allow for four displays to be populated based on a singular 16-bit uint16_t input
+ * as well as 4 individual 8-bit uint8_t inputs.
+ * 
+ * How To Use Module: Call SevenSeg_Init() to initialize the module.
```
Comment (muhlbaier) [inline include/seven_seg.h:14]: consider adding example code using @code and @endcode tags

File: include/seven_seg.h [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * @param seg2 number to be displayed in the third display from the left
+ * @param seg3 number to be displayed in the right most display
+ *
+ * Example: SevenSeg_DisplayRaw(1, 5, 8, 2); => will display as "1582" on seven segment display
+ *
+ */
+void SevenSeg_DisplayRaw(uint8_t seg0, uint8_t seg1, uint8_t seg2, uint8_t seg3);
+
+ /** @} */
+#endif /* _SEVEN_SEG_H_ */
```
Comment (muhlbaier) [inline include/seven_seg.h]: missing new line at end of file

**Review by muhlbaier — CHANGES_REQUESTED** [review]
Great job overall just needs a couple tweaks. Also, ask the expected users if they will need to show decimal numbers (if yes will they ever be negative?) and if so add a method to display a number in decimal or add a helper function to do a binary to BCD conversion.

**Comment by msorce26** [comment]
Requested changes have been made to seven_seg.h. Files seven_seg.c, hal_seven_seg.h, and hal_seven_seg.c have been added.

**Review by muhlbaier — APPROVED** [review]
Good work

**Pull request merged**